macapp/App/Sources/App/WebViewController.swift

```diff
@@ -66,7 +66,7 @@ class WebViewController<State, Action>:
     view = self.webView
   }
 
-  nonisolated func userContentController(
+  func userContentController(
```

Comment:
another example of something that was always safe, but just needed apple to correctly audit and annotate their own code.